### PR TITLE
optional height and width arguments to the append_plot method

### LINKED ISSUE #99: append_plot with additional (optional) width and height arguments
We have to add optional width and height arguments to the append_plot method.
So we could inherit the size from tm module arguments.

### PR DESCRIPTION
**Pull request by Polkas**
closes #99 

add optional height and width arguments to the append_plot method of the ReportCard

### CODE REVIEW

File: tests/testthat/test-ReportCard.R [hunk 1]
```diff
@@ -39,6 +39,13 @@ testthat::test_that("append_plot accepts a ggplot", {
   )
 })
 
+testthat::test_that("append_plot accepts a ggplot with a dim", {
+  testthat::expect_error(
+    ReportCard$new()$append_plot(ggplot2::ggplot(iris), c(1000L, 100L)),
+    regexp = NA
+  )
+})
+
```
Comment (mhallal1) [inline tests/testthat/test-ReportCard.R:48]: please add a test case with no dim added (what is the expected output) and with wrong dim input.

File: tests/testthat/test-PictureBlock.R [hunk 2]
```diff
@@ -33,6 +33,26 @@ testthat::test_that("set_content returns the PictureBlock object", {
     block$set_content(ggplot2::ggplot(iris)),
     block
   )
+  testthat::expect_identical(
+    dim(png::readPNG(block$set_content(ggplot2::ggplot(iris))$get_content())),
+    c(600, 800, 4)
+  )
```
[the remark this answers is not in the document]
Comment (Polkas) [reply tests/testthat/test-PictureBlock.R:39]: yes there is a need, as here we are checking the default setup

[the remark this answers is not in the document]
Comment (Polkas) [reply tests/testthat/test-PictureBlock.R]: yes, there was a duplication.

**Review by mhallal1 — APPROVED** [review]
tested with https://github.com/insightsengineering/teal.widgets/pull/63#pullrequestreview-1028328247 and working as expected.

**Pull request merged**